src/core/client/base.js

```diff
@@ -16,6 +17,8 @@ import log from 'core/logger';
 
 
 export default function makeClient(routes, createStore) {
+  FastClick.attach(document.body);
```

Comment:
I would like to see this added after the Sentry logging so that we can get reports of FastClick errors :)

While you're in here, could you put a comment above the Sentry configuration saying something like "This should come before any other code to catch and report errors" ?